packages/core/src/agent/TransportService.ts

```diff
@@ -18,6 +19,15 @@ export class TransportService {
     return Object.values(this.transportSessionTable).find((session) => session?.connectionId === connectionId)
   }
 
+  public setConnectionIdForSession(sessionId: string, connectionId: string) {
+    const session = this.findSessionById(sessionId)
+    if (!session) {
+      throw new AriesFrameworkError(`Session not found with id ${sessionId}`)
+    }
+    session.connectionId = connectionId
+    this.saveSession(session)
```

Comment:
We directly mutate the session so this doesn't really do anything? But for completeness it makes sense